src/main/kotlin/org/rucca/cheese/model/PatchTaskMembership200ResponseDataDTO.kt

```
@@ -0,0 +1,13 @@
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.Valid
+
+/** @param participant */
+data class PatchTaskMembership200ResponseDataDTO(
+    @field:Valid
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("participant", required = true)
+    val participant: TaskMembershipDTO
+) {}
```

detekt

[warning] The class or object PatchTaskMembership200ResponseDataDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)